docs/features/wired-mesh.rst

```diff
@@ -85,3 +85,33 @@ Disable Mesh-on-LAN::
 Please note that this configuration has changed in Gluon 2016.1. Using
 the old commands on 2016.1 and later will break the corresponding options
 in the *Advanced settings*.
+
+Mesh on other interfaces
+========================
+In some cases, you may want to mesh on more interfaces than only LAN or WAN.
+
+This is especially useful if you have multiple other nodes connected to a central node,
+and don't want them to mesh with each other, but only the central node.
+
+In this case, you may need to isolate them via VLANs on the device switch.
+
+Starting from Gluon 2020.2 it's possible to configure additional mesh-interfaces
+by adding an interface configuration with the proto ``gluon_wired``::
+
+  uci set network.mesh_add=interface
+  uci set network.mesh_add.proto=gluon_wired
+  uci set network.mesh_add.index=7
```

Comment:
I think it would be a good idea to use a index that is unlikely to cause conflicts for this example; 7 conflicts with the mesh VPN.

In the current setup of Gluon, the indices 2 and 6 are unused for mesh interfaces and should always be safe to use.